can_bus/quadruna/VC/VC_alerts.json

```diff
@@ -20,6 +20,10 @@
         "MissingFSMHeartbeat",
         "MissingDIMHeartbeat",
         "LeftInverterFault",
-        "RightInverterFault"
+        "RightInverterFault",
+        "LEStopFault", 
+        "REStopFault",
+        "TSMSFault",
+        "PCMInterlockFault"
```

Comment:
Can you make these warnings instead? In theory this doesn't matter because if any of these faults are set, the shutdown loop is open, and so we don't have HV and obviously can't drive.

However, with any sensor, there is always the possibility of sensor error, transients, noise, etc. which throw off your measurement. If that happened while driving, we wouldn't want to unnecessarily throw a fault and shutdown the car. 

On the other hand, if there is a true shutdown loop fault, the code doesn't need to do anything to make the car safe, because the shutdown loop will have already opened. So, there's really no downside of just making these warnings.

Reply:
(For anyone reading this who doesn't know, faults cause the vehicle to exit drive state (no torque from motors), and warnings don't do anything (just for logging))

Reply:
Makes sense, moved to warnings.